src/server/views/dashboard/lists-item.njk

```diff
@@ -50,27 +50,7 @@
         </div>
       </form>
 
-      <script nonce="{{cspNonce}}">
-        const hiddenClass = "govuk-radios__conditional--hidden";
```

Comment:
This is rolled up with govuk-frontend. We just weren't initialising the scripts properly.